packages/ui/src/FormGroupHeading.tsx

```diff
@@ -24,14 +32,17 @@ export const FormGroupHeading: React.FC<Props> = ({
       p={2}
       borderRadius={8}
       w='100%'
-      onClick={onClick}
+      tabIndex={1}
+      cursor='pointer'
     >
       {icon && <Icon width='20px' height='20px' mr={2} as={icon} />}
       <Heading fontSize='md' lineHeight='20px' fontWeight='700'>
         {title}
       </Heading>
       <Spacer />
-      {chevronIcon}
+      <Button onClick={onClick} tabIndex={1} variant='ghost' color='black'>
```

Comment:
Buttons don't need a tabIndex, they're tabbable by default.

Reply:
Choosing to use the Chakra `<Button>` component added some new styling here that adds padding that wasn't there before. Maybe this is desirable, but it's a change worth considering.

Reply:
While I think the padding unintentionally introduced by the button is pleasant, the button adds a tab stop that's redundant to the functionality of the form group header and we should not introduce it

Reply:
in case we decide to keep the button, it should also have an `onKeydown` listener

Reply:
`onClick` captures the typical accessibility keys for proceeding (space bar, enter etc)

Reply:
Sorry, I was mistaken. That is only the case if you use proper elements. Even adding accessibility attributes like `tabIndex` and `role` don't work if it's not a `button` or `a`: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role#required_javascript_features

Although for what it's worth, that's a reason to get the element to use the right element instead of using a div and trying to turn it into an accessible clickable element. This would be a lot easier if it was just a `<button>`.

Reply:
Refactoring this to Button element seems to work just fine. No redundant functionality with tab stop, and open/close works for both onClick and keydown (Enter and Space bar) without explicitly specifying this.

Reply:
[refactor to button element.webm](https://github.com/fedimint/ui/assets/41724425/4b0609d2-1363-4705-b241-44e974173c34)